app/charts/shared/chart-state.ts

```diff
@@ -0,0 +1,8 @@
+import { ChartType } from "@/configurator";
+
+import { Bounds } from "./use-width";
+
+export interface CommonChartState {
+  chartType: ChartType;
+  bounds: Bounds;
+}
```

Comment:
Could allData and preparedData be here ?

Reply:
Yes, that was my goal 😅 But as always, maps are different here and there we have data for areas and data for symbols... That's why allData and preparedData are not here yet, I need to think about how to handle this.